app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<p style="color: green"><%= notice %></p>
+
+<div class="container w-75">
+  <h2 class="text-center">
+    <%= @recipe.name %>
+  </h2>
+  <p>
+    <strong>Preparation time:
+      <%= @recipe.preparation_time %>
+    </strong>
+  </p>
+  <p>
+    <strong>Cooking time:
+      <%= @recipe.cooking_time %>
+    </strong>
+  </p>
+  <p>
+    <strong>Steps go here ... </strong>
+  </p>
+
+  <%= link_to "Generate shopping list", general_shopping_lists_index_path %> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 
+  <% if can? :manage, @recipe %>
+    <%= link_to "Add ingredients", new_food_path %>
```

Comment:
- [ ] Kindly check the add ingredients button it takes you to a new food item. see the wireframe you should be able to add ingredients to the recipe
![image](https://user-images.githubusercontent.com/48371156/198843855-24c44ca7-5fc2-4500-be36-bd5f9cd72d90.png)